kommunicate/src/main/java/io/kommunicate/Kommunicate.java

```diff
@@ -30,6 +31,11 @@
 import com.applozic.mobicommons.commons.core.utils.Utils;
 import com.applozic.mobicommons.json.GsonUtils;
 import com.applozic.mobicommons.people.channel.Channel;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.reflect.TypeToken;
+
+import org.json.JSONObject;
```

Comment:
@shubhamtewari is this getting used?

Reply:
@reytum is we are using Gson then do we need to use org.json.JSONObject as well?

Reply:
we dont require the import, removed it